src/styles/blocks/icon.scss

```diff
@@ -0,0 +1,9 @@
+.icon {
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+  &--cross {
+    height: 13px;
```

Comment:
Add 'background-image' property to 'icon--cross' with a proper url or base64 for the cross image.